Analysis/Stan/mapping_model_generate.stan

```diff
@@ -37,9 +37,11 @@ data {
   int <lower=1> N_edges;       // number of edges between grid cells
```

Comment:
There were lots of changes logged but I think the only real changes were related to the posterior predictive checks (and the od_param change).

Reply:
Yes that's is